base/server/upgrade/10.11.0/01-FixChallengePasswordClassPath.py

```diff
@@ -0,0 +1,60 @@
+# Authors:
+#     Christina Fu <cfu@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright (C) 2015 Red Hat, Inc.
```

Comment:
We still have a couple more days left in 2020 :)

Reply:
I'd suggest to use the simpler license header like [this](https://github.com/dogtagpki/pki/blob/master/base/server/upgrade/10.10.2/01-AddProfileCaAuditSigningCert.py) for new files.